src/i18n/es.js

```diff
@@ -93,7 +94,7 @@ Encontrarás un Email con los pasos a seguir.`,
     copyright_MD: `© ${year} Doppler LLC. Todos los derechos reservados. [Política de Privacidad y Legales](${urlPrivacyFromLogin}).`,
     enter_doppler: `Ingresa a tu cuenta`,
     enter_doppler_sub: `¡Hoy es un buen día para potenciar tu negocio con el poder del Email, Automation & Data Marketing!`,
-    error_payment_MD: `La cuenta está bloqueda, por favor [contáctate con Soporte](${mailtoSupport}).`,
+    error_payment_MD: `¡Ouch! Esta cuenta ha sido suspendida por falta de pago.\n\nPor favor [contacta al equipo de Atención al Cliente](${mailtoSupportBlockedAccount}) para solucionarlo.`,
```

Comment:
It is fine, but what do you think about storing the messages with interlines in a more friendly way?

```suggestion
    error_payment_MD: `
¡Ouch! Esta cuenta ha sido suspendida por falta de pago.

Por favor [contacta al equipo de Atención al Cliente](${mailtoSupportBlockedAccount}) para solucionarlo.`,
```

Reply:
Maybe I am doing something wrong but it seems to be not too friendly with that format:
![image](https://user-images.githubusercontent.com/10213170/87704111-3f466a80-c772-11ea-9c03-d191ba38ce17.png)